packages/console/src/utils/connector-form.ts

```diff
@@ -6,9 +6,19 @@ import { conditional } from '@silverhand/essentials';
 import { SyncProfileMode, type ConnectorFormType } from '@/types/connector';
 import { safeParseJson } from '@/utils/json';
 
+/**
+ * @remarks
+ * - When creating a new connector, it's called in the `convertFactoryResponseToForm()` method. At this time, there is no `config` data, so the default values in `formItems` are used.
```

Comment:
"it's called" sounds ambiguous. maybe "this function will be"

Reply:
updated